Dockerfile

```diff
@@ -14,19 +14,20 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
     org.label-schema.version="latest"
 
 # Installs latest Chromium package.
+# libstdc, harfbuzz, nss, freetype, are deps in chromium wqy-zenhei does not exist, not sure what this does
```

Comment:
Not sure what this means.

Reply:
libstdc, harfbuzz, nss, freetype, are deps in chromium you used to have them explicitly declared which did nothing except make everything more brittle. https://github.com/Zenika/alpine-chrome/blob/8c9e5217db6ea718a76d4824c96cc97d70c676b6/Dockerfile#L17

"wqy-zenhei" was renamed in alpine. you fixed that here https://github.com/Zenika/alpine-chrome/commit/2b7717a82b6e164dbf95f98add7c5d957d3942c4

Reply:
@zigarn they would have been removed for five months earlier if this was merged. =)